gobblin-data-management/src/main/java/org/apache/gobblin/data/management/conversion/hive/entities/StageableTableMetadata.java

```diff
@@ -92,6 +98,12 @@
    */
   public static final String SOURCE_DATA_PATH_IDENTIFIER_KEY = "source.dataPathIdentifier";
 
+  /**
+   * Attributes like "avro.schema.literal" are usually used in offline system as the source-of-truth of schema.
+   * This configuration's value should be the key name that users expects to preserve to schema string if necessary.
+   */
+  public static final String SCHEMA_SOURCE_OF_TRUTH = "schema.sourceOfTruth";
```

Comment:
Call it "schema.original"? schema.sourceOfTruth refers to the source where you obtained the schema from, which does not look like how you are setting this property.